### PR TITLE
feat(search): Added search page - FRONT-2348

### PR DESCRIPTION
**Pull request by tibi2303**
The multiselect will be added after the Multiselect PR will be accepted and merged.

### CODE REVIEW

File: src/compositions/search/search.html.twig [hunk 1]
```diff
@@ -0,0 +1,62 @@
+{% spaceless %}
+
+<header>
+  {% include '@oe-bcl/navbar/navbar.html.twig' with data.navbar only %}
+</header>
+<main>
+  <div class="container mt-5">
```
Comment (papegaill) [inline src/compositions/search/search.html.twig:7]: I think we need to discuss this at team level, for me we would need a general class like `.bcl-search` and not use utility classes for spacing and styles, this should be defined in a dedicated sass file. cc: @planctus @emeryro 
More classes should be added for the sub-element like title etc..  to be able to target and style them directly via sass..

File: storybook/.storybook/preview-head.html [hunk 2]
```diff
@@ -51,6 +51,42 @@
...
+          window.innerWidth || document.documentElement.clientWidth;
+      };
+
+      // Log the viewport width into the console
+      var logWidth = function () {
+        if (viewportWidth < 992) {
+          offcanvasMobile.classList.add("offcanvas", "offcanvas-start");
+        } else {
+          offcanvasMobile.classList.remove("offcanvas", "offcanvas-start");
+          offcanvasMobile.style.visibility = "visible";
```
Comment (papegaill) [inline storybook/.storybook/preview-head.html:73]: I think we should add `bsOffcanvas.hide();` here to ensure the offcanvas closed if resizing to desktop

File: storybook/.storybook/preview-head.html [hunk 3]
```diff
@@ -51,6 +51,42 @@
...
+
+      // On resize events, recalculate and log
+      window.addEventListener(
+        "resize",
+        function () {
+          setViewportWidth();
+          logWidth();
+        },
+        false
+      );
```
Comment (papegaill) [inline storybook/.storybook/preview-head.html:89]: do we have a plan of how this will be shiped .. or do we create a new ticket for this?

**Review by papegaill — COMMENTED** [review]
Nice work 👍 here are some remarks:

The close btn icon is missing on the offcanvas:
Screenshot 2021-06-24 at 18 48 46
Horizontal padding missing on the offcanvas for tablet format:
Screenshot 2021-06-24 at 18 46 26
Filter option icon is not visible:
Screenshot 2021-06-24 at 18 45 47
Content is cut within the sidebar, due to `.offcanvas-body `class (this class needs to be toggle in js, I guess?)..
Screenshot 2021-06-24 at 18 32 57

File: storybook/.storybook/preview-head.html (hunk 2 above)
Comment (papegaill) [reply storybook/.storybook/preview-head.html:73]: I think we should rename variable and classes names to something related to `bcl-search` as a prefix

**Review by papegaill — COMMENTED** [review]
seems to be a style issue on all stories when viewing in fullscreen via the top left icon cc:@planctus

File: src/components/badge/badge.html.twig [hunk 4]
```diff
@@ -6,6 +6,7 @@
   - url (string) (default: '')
   - title (string) (default: label)
   - hidden_text (string) (default: '')
+  - dismissible (boolean) (default: false)
```
[the remark this answers is not in the document]
Comment (tibi2303) [reply src/components/badge/badge.html.twig:9]: I added a control for dismiss in badge.

**Comment by papegaill** [comment]
@tibi2303 FYI since lastest changes, it seems that the search preview is broken.

**Comment by tibi2303** [comment]
@papegaill should work now, i had updated the data accordingly but it had a conflict in badge, probably you will need to add the modifications in patterns after merge also, I added dismissible param.

**Review by papegaill — APPROVED** [review]

**Pull request merged**